stories/core/Table.stories.tsx

```diff
@@ -1273,6 +1274,8 @@ export const Full: Story<Partial<TableProps>> = (args) => {
     [],
   );
 
+  const hasColumnManager = true;
```

Comment:
looks weird... why you need this variable?
Just pass true directly on line 1318